server/classes/Game.ts

```diff
@@ -0,0 +1,15 @@
+import Player from "./Player";
+import Room from "./Room";
+
+export class Game {
+    id: string;
```

Comment:
Volontaire que l'id soit un string ?

Reply:
Yep, IDs are UUIDs. This can be changed quite easily, though, as we have an IdGeneratorService.